Integrations/CentralBankIntegrationLib/CentralBankNewsPublisher.cs

```diff
@@ -20,8 +19,19 @@ public string GetNews()
             List<Valute> valutes = GetTopValutes();
 
             var sb = new StringBuilder();
-            sb.AppendLine("Top - 5 exchange rates:");
-            sb.AppendLine(valutes != null ? string.Join('\n', valutes) : Errors.CanNotGetCurrencyRateFromCB);
+            if (valutes.Count == 0)
+            {
+                sb.AppendLine("Valutes are not founded");
+            }
+            else
+            {
+                sb.AppendLine("Top - 5 exchange rates:");
+                foreach (Valute valute in valutes)
+                {
+                    var description = valute.Description();
+                    sb.AppendLine(description);

```

Comment:
I suppose the _Why not use var_ comment by Maksim wasn't about this. 
`sb.AppendLine(valute.Description());` must be okay, meanwhile `foreach (var valute in valutes)` better fits out convention

Reply:
fixed